generative_data_prep/processors/article_tokenizer.py

```diff
@@ -288,6 +294,10 @@ def tokenize(self, completion: str,
         token_type_ids: List[int] = []
 
         if prompt:
+            if self.prompt_prefix:
```

Comment:
If you make the default `""` then you can always just do `prompt = self.prefix_prompt + prompt + self.postfix_prompt`.  Your choice though